<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QWidget" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Giriş Arayüzü</string>
  </property>
  <widget class="QTabWidget" name="LoginTabWidget">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>170</y>
     <width>251</width>
     <height>226</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="DoktorGirisTab">
    <attribute name="title">
     <string>Doktor Girişi</string>
    </attribute>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lbldoktorid">
       <property name="text">
        <string>&amp;Doktor Id</string>
       </property>
       <property name="buddy">
        <cstring>leDoktorId</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lbldoktorsifre">
       <property name="text">
        <string>&amp;Şifre</string>
       </property>
       <property name="buddy">
        <cstring>leDoktorSifre</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="leDoktorSifre"/>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QPushButton" name="BtnDoktorGiris">
       <property name="text">
        <string>Doktor Girişi</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="leDoktorId"/>
     </item>
     <item row="3" column="0">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="YoneticiGirisTab">
    <attribute name="title">
     <string>Yönetici Girişi</string>
    </attribute>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="lblyoneticikadi">
       <property name="text">
        <string>&amp;Yönetici Id</string>
       </property>
       <property name="buddy">
        <cstring>leDoktorId</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="leYoneticiKullaniciAdi"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblyoneticisifre">
       <property name="text">
        <string>&amp;Şifre</string>
       </property>
       <property name="buddy">
        <cstring>leDoktorSifre</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="leYoneticiSifre"/>
     </item>
     <item row="2" column="0">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>38</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QPushButton" name="BtnYoneticiGiris">
       <property name="text">
        <string>Yönetici Girişi</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>38</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>LoginTabWidget</tabstop>
  <tabstop>leDoktorId</tabstop>
  <tabstop>leDoktorSifre</tabstop>
  <tabstop>BtnDoktorGiris</tabstop>
  <tabstop>leYoneticiKullaniciAdi</tabstop>
  <tabstop>leYoneticiSifre</tabstop>
  <tabstop>BtnYoneticiGiris</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
